From 5018ca7d9ce64b8cd11e39a69575e7d2dec84554 Mon Sep 17 00:00:00 2001
From: Jared Morrow <jm@joyent.com>
Date: Fri, 21 Jul 2017 17:51:59 -0600
Subject: [PATCH] MANTA-2900 mlocate should not need to setupMantaBuckets
 Reviewed by: David Pacheco <dap@joyent.com> Approved by: David Pacheco
 <dap@joyent.com>

---
 bin/mlocate | 1 +
 1 file changed, 1 insertion(+)

diff --git a/bin/mlocate b/bin/mlocate
index d592789..993e949 100755
--- a/bin/mlocate
+++ b/bin/mlocate
@@ -81,6 +81,7 @@ function createMorayClient(opts, cb) {
 
     var log = LOG.child({component: 'moray'}, true);
     opts.log = log;
+    opts.readOnly = true;
 
     var client = new libmanta.createMorayClient(opts);
 
-- 
2.21.0

